.github/workflows/docker-stable.yml

```diff
@@ -15,14 +15,20 @@ on:
 
 jobs:
   get-environment:
+    if: github.repository == 'centreon/centreon'
     uses: ./.github/workflows/get-environment.yml
     with:
       version_file: centreon/www/install/insertBaseConf.sql
 
   dockerize-stable:
     needs: [get-environment]
     runs-on: ubuntu-24.04
-    if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.get-environment.outputs.stability == 'stable' && github.ref_name == 'master' }}
+    if: |
```

Comment:
Shouldn t this also have a condition on internal-contributor ?

Reply:
Isn't this image used to run the tests ?

Reply:
it is intended to be built on release only, to be used by test runs as a "stable" reference. only internal should be able to run this in any situation